src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -28,42 +24,63 @@ our @EXPORT = qw();
 our @EXPORT_OK = qw( make_figure_of_merit );
 our %EXPORT_TAGS = ( all => \@EXPORT_OK );
 
+use Inline Python => q{
+import sys
+import perl
+# this provides an interface back to the Perl namespace
+
+try:
+    from starcheck.star_probs import acq_success_prob, prob_n_acq
+except ImportError as err:
+    # write errors to starcheck's global warnings and STDERR
+    perl.warning("Error with Inline::Python imports \n".format(err))
```

Comment:
warning() is that function in starcheck.pl in the original use of this code, and warning() isn't defined in FigureOfMerit.pm for this to do anything.  Do we want to try to catch import errors in general at this point or should starcheck just exit if we have trouble loading Python components?